newIDE/app/src/ResourcesList/index.js

```diff
@@ -79,6 +83,38 @@ export default class ResourcesList extends React.Component<Props, State> {
     this.props.onDeleteResource(resource);
   };
 
+  _scanForNewResources = () => {
+    const resourcesList = this
```

Comment:
Don't keep a reference to this, it's not necessary in ES6 thanks to arrow functions (`()=> {}`) :)
Instead just call `this.forceUpdate();` as the end. Will make things more consistent to always use this, and easier to search in the codebase for `this.forceUpdate();` :)

Reply:
I initially did use just this.forceUpdate() instead, but that crashed the newIde with an error about anonymous function for some reason. Should I call it with a different syntax?

Reply:
No, keep `this.forceUpdate`. This is because you're using `function` below, it should be an arrow function (see my other comment).